src/calliope/backend/latex_backend_model.py

```diff
@@ -45,7 +45,7 @@ def build(self, include: Literal["all", "valid"] = "all", **kwargs) -> None:
             **kwargs: kwargs for the LaTeX backend.
         """
         backend = LatexBackendModel(self._inputs, include=include, **kwargs)
-        backend._build()
+        backend.build()
```

Comment:
Changed this to a public since we do access it outside of this file. Can revert if needed.

Reply:
It may make sense. The reason I had it hidden was because we don't want `model.backend.build()` to be available to the user

Reply:
Hmmm... I tend to prefer viewing each module by its own "scope" (i.e., `model` is aware of `backend`, but not the other way around). This makes it clear what is meant to be used at the file level, and it's a bit less tangly in the long-run.

If we want to avoid users messing with the backend, then it's `model._backend` what should change (with us providing interfaces / wrappers). But honestly I think that even in that case it is not needed.

Reply:
In theory, an advanced user could create an additional "step" that injects stuff and re-builds the backend... and it may work!

However, if we really want to "hide" this function, I could move it to the `__init__.py` get function, and just return the fully built backend (no difference).
That way, all the `_build()` usage stays within the backend module, somewhat.

Reply:
We had `_backend` and an interface to that in the previous version and it was a real pain to maintain. We could simply rename `_build` to something more exlicit like `add_all_math`

Or, since we never have any intermediate steps between backend model initialisation and calling `_build`, we could move the `_build` step to within `BackendModel.__init__`. I think that's my prefered option.

Reply:
I'll do the following:
- Rename `backend.build` -> `backend.build_all_math` (in theory it's added during `__init__`, so I like this name more.
- Move `backend.build` to the `__init__` function so that upper functionality has no access to it.
- Add a `build_math` boolean to the `__init__` function (defaults to `True`) to deactivate math building in cases were we just want the backend.

Reply:
I am doing this because, for some test cases, we only want the backend, and we inject math into it.